fullthrottlelabs/reapop
# fix slide transition crash

### Related issues:  
### Changes
 
 
- Added explicit from and to transition objects to SlideTransition component

### Testing guide
 
1. On Chromium <= 83 based browsers, create a toast message and close it.
2. It should not crash the page


### Checklist
- [x] I ran the tests with `npm run test` (a few snapshot tests failed but that is expected)
- [ ] I added tests for the bug I fixed (if applicable)
- [ ] I updated the README or the DOCUMENTATION (if applicable)
- [ ] I updated the demo (if applicable)
- [ ] I verified my changes on all browsers listed below:
